x/oracle/proposal_handler.go

```diff
@@ -0,0 +1,33 @@
+package oracle
+
+import (
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	sdkerrors "github.com/cosmos/cosmos-sdk/types/errors"
+	govtypes "github.com/cosmos/cosmos-sdk/x/gov/types"
+	"github.com/medibloc/panacea-core/v2/x/oracle/keeper"
+	"github.com/medibloc/panacea-core/v2/x/oracle/types"
+)
+
+func NewOracleProposalHandler(k keeper.Keeper) govtypes.Handler {
+	return func(ctx sdk.Context, content govtypes.Content) error {
+		switch c := content.(type) {
+		case *types.OracleUpgradeProposal:
+			return handlerOracleUpgradeProposal(ctx, k, c)
+		default:
+			return sdkerrors.Wrapf(sdkerrors.ErrUnknownRequest, "unrecognized oracle upgrade proposal content type: %T", c)
+		}
+
+	}
+}
+
+func handlerOracleUpgradeProposal(ctx sdk.Context, k keeper.Keeper, p *types.OracleUpgradeProposal) error {
```

Comment:
We can check if current block height is less than the target height of upgrade as upgrade keeper's handler do.
https://github.com/cosmos/cosmos-sdk/blob/fa2cb40892e4653926dd8ff9fb6702e80313af33/x/upgrade/keeper/keeper.go#L186